test/replication-luatest/gh_7581_downstream_lag_test.lua

```diff
@@ -30,6 +30,7 @@ g.before_each(function(g)
         box_cfg = box_cfg,
     })
     g.cluster:start()
+    g.cluster:wait_for_fullmesh()
```

Comment:
AFAIU this can only fail when `server2` becomes the bootstrap leader. Then `server1` has to sync first.
Let's better make server2 never become the bootstrap leader.

Reply:
Never mind, let's keep it as is.